ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -718,6 +721,41 @@ def trunc(x):
     return ivy.trunc(x)
 
 
+@with_supported_dtypes(
+    {"0.4.14 and below": ("float32", "float64", "int32", "int64")}, "jax"
+)
+@to_ivy_arrays_and_back
+def unwrap(p, discont=None, axis=-1, period=2 * ivy.pi):
+    p = ivy.asarray(p)
```

Comment:
Is there a reason for the explicit conversion happening here?

Reply:
@bipinKrishnan - I implemented this as per the JAX source code [here](https://jax.readthedocs.io/en/latest/_modules/jax/_src/numpy/lax_numpy.html#unwrap)